docs/templates.md

```diff
@@ -47,6 +47,8 @@ The following functions are defined.
   the content. The `label` is evaluated as a space-separated string.
 * `if(condition: Boolean, then: Template[, else: Template]) -> Template`:
   Conditionally evaluate `then`/`else` template content.
+* `coalesce(content: Template...) -> Template`: Returns the first **non-empty**
+  content.
```

Comment:
I prefer easily readable expressions. Since a VCS is used mostly by programmers, it seems okay to use operators, but at the same time, they are all writing in languages with different syntax and it would be best not to introduce yet another set of operators for the VCS. (I already don't like the operators for ancestors and children since it's more to memorize, just to use a tool.)